### PR TITLE
Binary type

### PR DESCRIPTION
**Pull request by iOiurson**
I chose to represent the mesh type differentiation by changing their material color.

### CODE REVIEW

**Comment by iOiurson** [comment]
Oh, and this is to solve #77.

File: front/src/createBuildingMesh.js [hunk 1]
```diff
@@ -13,8 +13,25 @@ function zFrom8bitsInt(z, MIN_Z, MAX_Z){
     return z*(MAX_Z - MIN_Z)/(((1 << 8)-1)) + MIN_Z;
 }
 
+var intToMeshType = {
```
Comment (DavidBruant) [inline front/src/createBuildingMesh.js]: Move this to their own file so it can be shared with 3dsFormatToAntsBinaryBuffer.js
Right now, 3dsFormatToAntsBinaryBuffer.js hardcodes the values (with associated comments) which isn't good as it may fall out of sync. If both share the mapping, no risk to fall out of sync.

File: src/3dsFormatToAntsBinaryBuffer.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ function encodeVertex(x, y, z){
 
     returns a Buffer encoded as :
     
-    object : | nbVertices(2) | v1(4) | v2(4) | ... | vn(4) | nbFaces(2) | f1(6) | f2(6) | ... | fn(6) |
+    object : | nbVertices(2) | v1(4) | v2(4) | ... | vn(4) | nbFaces(2) | f1(6) | f2(6) | ... | fn(6) | meshType(2)
```
Comment (DavidBruant) [inline src/3dsFormatToAntsBinaryBuffer.js]: Ok, so meshType is in the end.

Comment (iOiurson) [inline src/3dsFormatToAntsBinaryBuffer.js]: Yep, but maybe it would be better otherwise ?

Comment (DavidBruant) [inline src/3dsFormatToAntsBinaryBuffer.js]: I have no opinion to be honest. It was just a drive-by comment.
As long as it's clearly documented (maybe the comments we have now are enough), I don't think it makes a difference.
